### ISSUE TITLE
Link to docs in README is broken

### ISSUE THREAD

**Issue by shaftoe** [tehbilly/hugo-example-blog#1]
http://hugo.spf13.com/overview/introduction returns a 404:

```
$ curl -I http://hugo.spf13.com/overview/introduction
HTTP/1.1 404 Not Found
```

**Comment by tehbilly** [comment]
Good catch. Various other links to `hugo.spf13.com` exist as well, and need to be updated to `gohugo.io` instead.

This example is terribly old at this point, but I can at least go through and update the links at some point soon. Thanks for letting me know!

**Comment by shaftoe** [comment]
Happy to help. Can you point me to a more up-to-date article? I'd like to get started, especially I'd like to understand how templates and themes relate. Thanks!

**Comment by tehbilly** [comment]
Everything I know of would be on the main website [here](https://gohugo.io). I haven't used hugo in quite awhile, but the documentation on [themes](https://gohugo.io/themes) and [templates](https://gohugo.io/templates) look pretty extensive!
